### DISCUSSION TITLE
LazyVim now requires at least nvim 0.9?

### DISCUSSION THREAD

**Discussion by titoOdUA** [LazyVim/LazyVim#824] [Q&A]
After a recent update telescope stopped working for me. It gives me an error that it requires 0.9 version of nvim (I have 0.8.3)
I have tried to reinstall LazyVim but it didn't help.

After a small research, I found this https://github.com/nvim-telescope/telescope.nvim/pull/2499

Does LazyVim from now on would require 0.9? (as the telescope is a significant part of the package) or there would be some tweaks that would allow to use older versions?

**Comment by folke** [comment]
Good point. You can pin Telescope to an older version yourself, but I'll see to add this automatically.

**Comment by folke** [comment]
I just pushed a fix for this. Do understand that when running in Neovim 0.8.0, Telescope will never be updated.
